brownie-config.yaml

```diff
@@ -0,0 +1,66 @@
+project_structure:
+    build: build
+    contracts: contracts/vyper
+    interfaces: interfaces
+    reports: reports
+    scripts: scripts
+    tests: test
+
+networks:
+    default: development
+    development:
+        gas_limit: max
+        gas_buffer: 1
+        gas_price: 0
+        max_fee: null
+        priority_fee: null
+        reverting_tx_gas_limit: max
+        default_contract_owner: true
+        cmd_settings: null
+    live:
+        gas_limit: auto
+        gas_buffer: 1.1
+        gas_price: auto
+        max_fee: null
+        priority_fee: null
+        reverting_tx_gas_limit: false
+        default_contract_owner: false
+
+compiler:
+    evm_version: null
+    solc:
+        version: null
+        optimizer:
+            enabled: true
+            runs: 200
+        remappings: null
+    vyper:
+        version: null
```

Comment:
Does this imply to use the latest version?

Reply:
nope ser, it actually depends on the pragma used in the contract

This is taken from the [documentation](https://eth-brownie.readthedocs.io/en/stable/config.html#config-reports)

> The version of vyper to use. Should be given as a string in the format 0.x.x. If set to null, the version is set based on the contract pragma

Reply:
Perfect